python/hsfs/feature_view.py

```diff
@@ -884,6 +1473,84 @@ def train_validation_test_split(
         """
         Get training data from feature groups.
 
+        !!! example 
+            ```python
+            # get feature store instance 
+            fs = ...
+
+            # get feature view instance 
+            feature_view = fs.get_feature_view(...)
+
+            # get training data
+            X_train, X_val, X_test, y_train, y_val, y_test = feature_view.train_validation_test_split(
+                validation_size=0.3,
+                test_size=0.2
+            )
+            ```
+
+        !!! example "Time Series split"
+            ```python
+            # get feature store instance 
+            fs = ...
+
+            # get feature view instance 
+            feature_view = fs.get_feature_view(...)
+
+            # set up dates
+            start_time_train = '2017-01-01 00:00:01'
+            end_time_train = '2018-02-01 23:59:59'
+
+            start_time_val = '2018-02-02 23:59:59'
+            end_time_val = '2019-02-01 23:59:59'
+            
+            start_time_test = '2019-02-02 23:59:59'
+            end_time_test = '2020-02-01 23:59:59'
+            # you can also pass dates as datetime objects
+
+            # get training data
+            X_train, X_val, X_test, y_train, y_val, y_test = feature_view.train_validation_test_split(
+                validation_size=0.3,
+                test_size=0.2,
```

Comment:
if you are using event time split then this is not necessary. Is this even working?